src/content/structured/components/alerts/code.mdx

```diff
@@ -118,21 +114,21 @@ export const snippetsWithAction = [
   },
   {
     language: "React",
-    snippet: `<IcAlert heading="With action" message="This alert has an action.">
-    <IcButton slot="action" variant="secondary" appearance="dark" onClick={(ev) => console.log(ev)}>Button</IcButton>
+    snippet: `<IcAlert heading="Did you know?" message="Coffee is delicious.">
+    <IcButton slot="action" variant="secondary" appearance="dark" onClick={(ev) => console.log(ev)}>Yes</IcButton>
 </IcAlert>`,
   },
 ];
 
 <ComponentPreview snippets={snippetsWithAction}>
-  <IcAlert heading="With action" message="This alert has an action.">
+  <IcAlert heading="Did you know?" message="Coffee is delicious.">
```

Comment:
Think this might be better as something like this:
```suggestion
  <IcAlert heading="Did you know?" message="There are a new range of coffees to try.">
```